src/components/Endpoint/styled.elements.ts

```diff
@@ -1,4 +1,7 @@
+import { palette } from '@leafygreen-ui/palette';
 import styled from '../../styled-components';
+// import { HTTPBadgeColors } from '../../theme';
```

Comment:
Unneeded now with getBadgeStyles here, unless left as a commentary

Reply:
Thanks for catching! This was not intentional. Removed